cypresse2e/cypress/integration/e2eTests/offerHelp/offerHelpQuestion.spec.js

```diff
@@ -0,0 +1,109 @@
+import OfferHelpQuestionPage from '../../../elements/pages/offerHelpQuestionPage';
+
+
+describe('Fight Pandemics Offer Help Questionnaire', () => {
+
+    const offerHelpQuestion = new OfferHelpQuestionPage();
+
+
+    context('User goes to offer help question one page', () => {
+        beforeEach(() => {
+            offerHelpQuestion.visitOfferHelpOne();
+        });
+
+
+        it('Offer Help Question One page contains heading & question one', () => {
+            var questionNumberOne = "Question 1/3";
+            var questionOne = "How do you want to contribute?"
+            offerHelpQuestion.getQuestionNumberOne().should('be.visible').contains(questionNumberOne);
+            offerHelpQuestion.getQuestionOne().should('be.visible').contains(questionOne);
+        });
+
+        it('Offer Help Question One page contains answer options & checkboxes to select option/options for question one', () => {
+            var checkboxOptionOne = "As a Volunteer";
+            var checkboxOptionTwo = "As a Donor/Investor"
+            var checkboxOptionThree = "As an Organisation"
+            var checkboxOne
+            var checkboxTwo
+            var checkboxThree
```

Comment:
I don't think the "checkboxOne", "..Two", "...Three" variables are being used. Can they be removed?

Reply:
Sure, will remove them. Thanks Jessica.